tests/test_network_model_data.py

```
@@ -182,63 +120,84 @@
+    of the order of SQL calls in the code being tested.
+    """
+    db_data = request.param['db_data']
+    mock_con = MagicMock(name='mock_connection')
+    mock_cursor = MagicMock(name='mock_cursor')
     mock_con.cursor.return_value = mock_cursor
-    mock_execute = MagicMock()
-    mock_cursor.execute.return_value = mock_execute
-    mock_execute.fetchall.side_effect = request.param['data']
-    return mock_con, request.param['res']
 
+    def dispatcher(query, *args):
```

Ruff (0.14.1)

Missing return type annotation for private function `dispatcher`

(ANN202)